Test2

msg4marina opened the pull request.

msg4marina committed:
Test2

dzmitryrak commented on src/test/java/Test2.java:
```diff
@@ -0,0 +1,71 @@
+            driver.findElement(By.name("q")).sendKeys("1");
+            //нажать кнопку Update
+            driver.findElement(By.cssSelector("[value=Update]")).click();
+            assertEquals(117.70, 102.30, "Total price рассчитан неверно");
```
Здесь ты сравниваешь две статичные цифры 117.70, 102.30

Первая - это актуальный результат, ее нужно получить со страницы

msg4marina replied:
Done!

msg4marina committed:
Test2:
1. Useless information deleted;
2. String29: Code to find the autogenerated email has been changed;
3.String41:Code with clear() method has been changed;
4.Added Strings 44-49;

msg4marina committed:
Update Test2 1.1.

1. Useless information deleted;
2. String29: Code to find the autogenerated email has been changed;
3.String41:Code with clear() method has been changed;
4.Added Strings 44-49;
